usr/local/sbin/restic_backup.sh

```diff
@@ -22,8 +22,17 @@ RETENTION_MONTHS=18
 RETENTION_YEARS=3
 
 # What to backup, and what to not
-BACKUP_PATHS="/ /boot /home /mnt/media"
-BACKUP_EXCLUDES="--exclude-file /.backup_exclude --exclude-file /mnt/media/.backup_exclude --exclude-file /home/erikw/.backup_exclude"
+BACKUP_PATHS="/ /boot /home"
+[ -d /mnt/media ] && BACKUP_PATHS+=" /mnt/media"
+BACKUP_EXCLUDES="--exclude-file /.backup_exclude"
```

Comment:
This needs to be changed to /etc/restic/ since the other PR

Reply:
Good catch! I was nervous which PR you would accept so I made them all individual. I'll have this fixed in a moment...